relay-general/src/store/transactions/snapshots/relay_general__store__transactions__processor__tests__scrub_span_identifiers_and_apply_rules.snap

```diff
@@ -0,0 +1,75 @@
+---
+source: relay-general/src/store/transactions/processor.rs
+expression: event
+---
+{
+  "type": "transaction",
+  "transaction": "/transaction-name/hi",
+  "transaction_info": {
+    "source": "sanitized"
+  },
+  "timestamp": 1619420400.0,
+  "start_timestamp": 1619420341.0,
+  "contexts": {
+    "trace": {
+      "trace_id": "4c79f60c11214eb38604f4ae0781bfb2",
+      "span_id": "fa90fdead5f74053",
+      "op": "default",
+      "type": "trace"
+    }
+  },
+  "spans": [
+    {
+      "timestamp": 1597976393.471877,
+      "start_timestamp": 1597976393.461967,
+      "description": "POST http://example.com/remains/to-scrub/remains-too/1234567890",
+      "op": "http.client",
+      "span_id": "bd2eb23da2beb450",
+      "parent_span_id": "8f5a2b8768cafb4e",
+      "trace_id": "ff62a8b040f340bda5d830223def1d81",
+      "status": "ok",
+      "data": {
+        "description.scrubbed": "POST http://example.com/remains/*/remains-too/*"
+      }
+    },
+    {
+      "timestamp": 1597976393.471877,
+      "start_timestamp": 1597976393.461967,
+      "description": "GET http://example.com/another/url/is/intact",
+      "op": "http.client",
+      "span_id": "bd2eb23da2beb451",
+      "parent_span_id": "8f5a2b8768cafb4e",
+      "trace_id": "ff62a8b040f340bda5d830223def1d81",
+      "status": "ok",
+      "data": {}
+    },
+    {
+      "timestamp": 1597976393.471877,
+      "start_timestamp": 1597976393.461967,
+      "description": "POST http://example.com/remains/not-scrubbed-for-different-op/remains-too",
+      "op": "db.sql.query",
+      "span_id": "bd2eb23da2beb452",
+      "parent_span_id": "8f5a2b8768cafb4e",
+      "trace_id": "ff62a8b040f340bda5d830223def1d81",
+      "status": "ok"
+    }
+  ],
+  "_meta": {
+    "spans": {
+      "0": {
+        "data": {
+          "description.scrubbed": {
+            "": {
+              "rem": [
+                [
+                  "description.scrubbed:/remains/*/**",
```

Comment:
This doesn't really identify what span it applied to. Happy to update it, but I don't think this is really important right now.

Reply:
It has the `"0"` on line 59, doesn't that identify the span?